InventoryConfigurableProductIndexer/Indexer/SelectBuilder.php

```diff
@@ -8,17 +8,24 @@
 namespace Magento\InventoryConfigurableProductIndexer\Indexer;
 
 use Exception;
+use Magento\Catalog\Api\Data\ProductInterface;
 use Magento\Framework\App\ResourceConnection;
 use Magento\Framework\DB\Select;
+use Magento\Framework\EntityManager\MetadataPool;
+use Magento\InventoryCatalogApi\Api\DefaultStockProviderInterface;
+use Magento\InventoryIndexer\Indexer\IndexStructure;
+use Magento\InventoryIndexer\Indexer\InventoryIndexer;
```

Comment:
See https://github.com/magento-sparta/inventory/pull/168/files#diff-46d1eab5b946d8db6bc8da2f9f205df1b929013e6df65c0edd95b1af0f44de3bL16: you are missing the part that remove some "use".. because of this, some "use" is included twice:
- use Magento\InventoryIndexer\Indexer\IndexStructure;
- use Magento\InventoryIndexer\Indexer\InventoryIndexer;
- use Magento\Framework\EntityManager\MetadataPool;
 - use Magento\Catalog\Api\Data\ProductInterface;